src/AltinnCore/Common/Helpers/WorkflowHelper.cs

```diff
@@ -68,7 +68,7 @@ public static string GetUrlForCurrentState(Guid instanceId, string org, string a
                 case WorkflowStep.FormFilling:
                 case WorkflowStep.Submit:
                 case WorkflowStep.Archived:
-                    return $"/runtime/{org}/{appName}/{instanceId}/#Preview";
+                    return $"/runtime/{org}/{appName}/#/instance/{instanceId}";
```

Comment:
The url in "App" does not use /runtime/ in the path. Will this still work? @TheTechArch

Reply:
@TheTechArch  this was what we discussed a some time back about making runtime the default path in Altinn Studio and put gitea under a subfolder.  Now it will not work

Reply:
@TheTechArch : Jesper propose a temporary solution until we fix the gitea path issue: https://github.com/Altinn/altinn-studio/pull/2002#discussion_r295518192

Could you also create an issue related to the gitea rootfolder case?

Reply:
@jezpoz : #2029 should fix the path to Gitea in Altinn Studio, so no logic is necessary and no ```/runtime/``` path in Studio or Apps

Reply:
@matsgm Are you suggesting that I fix this before merging this then?

Reply:
If you can :)

Reply:
Implementing temporary checks for this. Since changing the whole routing within Altinn Studio is nothing to take lightly.